lib/v2/gov/radar.js

```diff
@@ -1008,6 +1008,14 @@ module.exports = {
                 target: '/gov/shenzhen/hrss/szksy/:caty/:page?',
             },
         ],
+        zjj: [
+            {
+                title: '深圳市住房和建设局',
+                docs: 'https://docs.rsshub.app/routes/government#guang-dong-sheng-ren-min-zheng-fu-shen-zhen-shi-zhu-jian-ju',
```

Comment:
```suggestion
                docs: 'https://docs.rsshub.app/routes/government#guang-dong-sheng-ren-min-zheng-fu-shen-zhen-shi-zhu-fang-he-jian-she-ju',
```
https://github.com/DIYgod/RSSHub/blob/e1c5ed21d16e5f154a97ebb55e4ccadaebfeee5f/website/docs/routes/government.md?plain=1#L235